writer_test.go

```diff
@@ -1273,7 +1273,7 @@ func ExampleMediaPlaylist_String() {
 // Create new media playlist
 // Add two segments to media playlist
 // Print it
-func ExampleMediaPlaylist_String_Winsize0() {
+func ExampleMediaPlaylist_String_winsize0() {
```

Comment:
Interesting one, `go test` wont even compile now because of this. Probs there's a way to skip checks but easier to just follow naming rules.